app/views/councillor_contributions/_contribution_form_input.html.haml

```diff
@@ -0,0 +1,15 @@
+%fieldset
+  %legend
```

Comment:
@equivalentideas you mean not to indent `%legend` under `%fieldset`?  I think `%legend` needs to be one level deeper indented under `%fieldset` to achieve this style
https://developer.mozilla.org/en/docs/Web/HTML/Element/fieldset 
I may be wrong as i am not very familiar with haml style.